src/pineko/evolve.py

```diff
@@ -354,7 +354,7 @@ def evolve_grid(
     evol_info = grid.evolve_info(order_mask)
     x_grid = evol_info.x1
     mur2_grid = evol_info.ren1
-    xif = 1.0 if operators1.operator_card.configs.scvar_method is not None else xif
+    xif = 1.0 if operators1.operator_card.configs.scvar_method is None else xif
```

Comment:
About this I'm not entirely sure I understand the logic... it has been there since a while (18 months)

Reply:
to be precise https://github.com/NNPDF/pineko/commit/f18c623496cb4fc9fa914d0b36960a3e65a49518 which is part of https://github.com/NNPDF/pineko/pull/82 - and I think this makes sense, because for scheme A and B eko does on it's own the necessary integral adjustments. Instead for scheme C pineko has to take care (as eko can not know); so this new changes seems to be wrong (same arguments apply of course above)

Reply:
About this I remember now the logic and I agree with @felixhekhorn . I started playing with this because I was getting an error from pineappl when producing the fktables but turns out that the other fix in this PR is enough to solve everything. So let me revert this change here